components/python/pytest/Makefile

```diff
@@ -42,18 +42,30 @@ include $(WS_MAKE_RULES)/prep.mk
 include $(WS_MAKE_RULES)/setup.py.mk
 include $(WS_MAKE_RULES)/ips.mk
 
+ASLR_MODE = $(ASLR_NOT_APPLICABLE)
+
+COMPONENT_POST_INSTALL_ACTION = \
+	for pv in $(PYTHON_VERSIONS); do \
```

Comment:
You have to do it just once for PYTHON_VERSION - COMPONENT_POST_INSTALL_ACTION will run for each of them, look for example, on components/python/fonttools/Makefile.